index.js

```diff
@@ -88,8 +88,8 @@ parser.parseName = function (name) {
         } else {
             attrs.firstName = parts.shift();
         }
-        
-        attrs.lastName = parts.pop();
+
+        attrs.lastName  = parts.length ? parts.pop() : ''; // set `lastName` to empty string if not provided
```

Comment:
can you remove the comment? its not necessary -- or put it on the line above.
code should not extend past 80 chars